pennylane_lightning/lightning_kokkos/lightning_kokkos.py

```diff
@@ -195,7 +195,7 @@ def __init__(
             shots=None,
             batch_obs=False,
             kokkos_args=None,
-        ):  # pylint: disable=unused-argument
+        ):  # pylint: disable=unused-argument, too-many-arguments
```

Comment:
Is this needed? Or was this added to keep it similar to ``lightning.gpu``? 
If this is not needed, I prefer to not add it.

Reply:
Removed! Thanks